forecast/paper/response-to-reviewers.tex

```diff
@@ -0,0 +1,304 @@
+  that our end 
+  date (31 December) coincides with the introduction of vaccinations.
+\end{resp}
+
+
+  3. This points to what is perhaps a more major issue. Can they identify in
+  quantitative terms the consequences of the additional improvements in
+  forecasting provided by these methods in terms of gains in resources available
+  to deal with the epidemic? Unfortunately as they state, in the crucial periods
+  of sharply rising cases, the additional benefit seems least.
+
+
+\begin{resp}
+  This is a good point, and we have made a few modifications described in more
+  detail below (Reviewer 2, Point 3).
+\end{resp}
+
+
+
+  4. Despite the fact that a detailed description of terms in the text is
+  impossible, it would I think be helpful to say if there is any significance to
+  the names they associate with their first 2 and 4th indicators.
+
+
+\begin{resp}
+  It seems that the Reviewer is referring to CHNG-COVID and DV-CLI. These are
+  described in the Methods Section, Signals and Locations Subsection. To be more
+  specific, Change 
+  Healthcare is a large healthcare insurance claims processor. The difference
+  between ``-CLI'' and ``-COVID'' is that the first measures insurance
+  claims related to symptoms
+  consistent with COVID while the second requires a medical diagnosis (through
+  testing or presumed positive by the medical provider). The ``DV'' modifier
+  corresponds to different claims processors (not Change Healthcare) who wish to
+  remain anonymous.
+
+  TODO Do we want to alter the text at all?
+\end{resp}
+
+
+
+  5. What are Change Healthcare claims..as opposed to others?
+
+
+\begin{resp}
+  Hopefully the discussion to point 4 above has cleared up any confusion.
+\end{resp}
+
+
+\section*{Reviewer 2}
+\begin{quote}
+  \emph{(items numbered by the editor for convenience of reference)}
+\end{quote}
+
+\subsection*{Statistical or Methodological Comments:}
+  
+1. Is this really a stationary phenomenon? Over the timerange extending from
+pre-lockdown to today, it certainly isn't. The method in this paper of
+presenting model prediction errors makes it impossible to see this. People in
+econometrics look at cumsums of normalized prediction errors. You can see change
+points and trends very easily.
+
+\begin{resp}
+  There is little reason to believe that this data is stationary. The models
+  however are trained and evaluated over reasonably short periods (2-3 months),
+  over which time the behavior may well be stationary. We have added a section
+  to the appendix that uses the cumulative mean as suggested (the sum is
+  difficult to see due to the scaling). With some exceptions at the beginning of
+  the evaluation period, the relative ordering remains fairly consistent. The AR
+  model is easily the worst.
+
+  TODO I have added the code and a caption for the figure to the Supplement, but
+  no corresponding 
+  text. It seems a fairly minor addition, not necessarily deserving its own
+  section. 
```

Comment:
From the plot, it looks like when all forecasters struggle, the AR stands out by struggling the most.  However, when all forecasters do well, AR does not stand out (even though it's still one of the worst).

Also, assuming that the shaded region is the US cases, it looks like these forecasters do best when things are linear (e.g., steady rise) and worst when there is curvature (peak and trough).